config.yml.example

```diff
@@ -10,12 +10,10 @@ script:
       - Cómo va, dude!? 🤖
   - type: dollar
   - type: random_gif
+    token: giphy_token
```

Comment:
I'm not sure about this change since this will force you to write `giphy_token` on every step with a GIF animation.

I know this is an edge case but stills rings a bell in my head :-)

Do you think we can change this?

Reply:
You have a point! Let me thing for something better.

Reply:
Have a look please at the new approach using a module to handle those configuration things.